basic/list1.py

```diff
@@ -34,7 +38,16 @@ def match_ends(words):
 # before combining them.
 def front_x(words):
     # +++your code here+++
-    return
+    list1=[]
```

Comment:
just a nitpick, but could you came up with a more meaningful names for these lists?